### PR TITLE
Add TimeTicket, ActorId, CRDTElement, Primitive, GRPCTypeAlias, YorkieError and related Tests

### PR DESCRIPTION
**Pull request by hyeongsik-won**
**What this PR does / why we need it**:

Add TimeTicket, ActorId, CRDTElement, Primitive, GRPCTypeAlias, YorkieError and related Tests

GRPCTypeAlias remap the protobuf type names locally.
- The Swift protobuf guide: https://github.com/apple/swift-protobuf/blob/main/Documentation/API.md#generated-struct-name

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation**:

```docs

```

**Checklist**:
- [ ] Added relevant tests or not required
- [ ] Didn't break anything

### CODE REVIEW

File: Sources/Document/Time/TimeTicket.swift [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+/**
+ * `TimeTicket` is a timestamp of the logical clock. Ticket is immutable.
+ * It is created by `ChangeID`.
+ */
+class TimeTicket {
+    private enum InitialValue {
+        static let initialDelimiter: UInt32 = 0
+        static let maxDelemiter: UInt32 = 4_294_967_295
```
Comment (hackerwins) [inline Sources/Document/Time/TimeTicket.swift:32]: Can we use a constant here? 🤔

```suggestion
        static let maxDelemiter: UInt32 = Uint32.max
```

File: Sources/Document/Time/TimeTicket.swift [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        static let maxLamport: UInt64 = 18_446_744_073_709_551_615
```
Comment (hackerwins) [inline Sources/Document/Time/TimeTicket.swift:33]: We can't represent uint64 in MongoDB, so we have to use `int64`.
https://github.com/yorkie-team/yorkie/issues/373

```suggestion
        static let maxLamport: Int64 = Int64.max
```

**Review by hackerwins — COMMENTED** [review]
Thanks for your contribution. Overall it looks good.
I leave a few comments.

File: Sources/Document/Time/TimeTicket.swift (hunk 2 above)
Comment (hyeongsik-won) [reply Sources/Document/Time/TimeTicket.swift]: I have fixed maxLamport and maxDelemiter types in TimeTicket.

**Pull request merged**